docs/reference/CLI.md

```diff
@@ -268,6 +269,10 @@ $ launchpad start greeter
                         This option can occur several times to configure more
                         than one settings file. Supported file settings formats
                         are INI and JSON.
+                --dry-run
+                        Perform a dry run of the application.
+                        All the configuration will be loaded (and validated),
+                        but the application will not perform any changes.
```

Comment:
Hmm... will not perform any changes. What do you mean? The application will not launch, if that's true make it more explicit.